src/routes/dictionaries/+page.svelte

```diff
@@ -125,6 +131,16 @@
                 required={isSequenceAdaptation}
               />
             </fieldset>
+          {:else if files !== undefined}
+            <fieldset class="save-dictionary-fieldset">
+              <label for="Save Dictionary">Save Command Dictionary to filesystem</label>
```

Comment:
Todo: change this label to say "Save Dictionary to filesystem" as we are going to update the backend to allow saving any kinds of dictionaries